app/src/main/res/layout/activity_cure_profile.xml

```diff
@@ -10,9 +10,20 @@
     tools:context=".Activity.Profile">
 
     <include
-        layout="@layout/cure_template"/>
+        android:id="@+id/include2"
+        layout="@layout/cure_template" />
 
     <include
         layout="@layout/bottom_sheet"/>
 
+    <ImageView
+        android:id="@+id/expanded_image"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```

Comment:
This won't look too attractive.
So make a change to it and show dialog instead of image view.

Reply:
OK